examples/system_tray.rs

```diff
@@ -7,27 +7,35 @@
 #[cfg(any(feature = "tray", all(target_os = "linux", feature = "ayatana")))]
 fn main() {
   #[cfg(target_os = "macos")]
-  use tao::platform::macos::{CustomMenuItemExtMacOS, NativeImage, SystemTrayBuilderExtMacOS};
   use tao::{
     event::Event,
     event_loop::{ControlFlow, EventLoop},
     menu::{ContextMenu as Menu, MenuItemAttributes, MenuType},
+    platform::macos::{CustomMenuItemExtMacOS, NativeImage, SystemTrayBuilderExtMacOS},
     system_tray::SystemTrayBuilder,
   };
```

Comment:
This breaks other platforms, because instead of:
only using `tao::platform::macos::*` on macos.
Now it:
only uses `tao::*` on macos.

And all of Tao is not imported.
Just revert to what it was and it works fine (tried on linux) :]

Reply:
@Beanow thanks, I have fixed it,  please refer to commit 4a53076.

Reply:
Resolves compile issues for me, though I don't have a mac environment ready. So I'll trigger the CI tests.